### PR TITLE
f-searchbox@v4.0.0-beta.20 - Fix dropdown flicker & display loading i…

### PR DESCRIPTION
**Pull request by kevinrodrigues**
### Added
- Spinner loading indicator for Loqate.
- Ability to hide dropdown when final address is picked.

### Fixed
- Drop down minor flicker when setTimeout is used.

### Video
https://www.loom.com/share/98bb00ed16b446ac99e7e88a47c06e6e

### CODE REVIEW

File: packages/components/molecules/f-searchbox/src/components/formElements/FormFullAddressContinueWithSuggestion.vue [hunk 1]
```diff
@@ -5,6 +5,7 @@
                 $style['c-continueWithSuggestion-btn'],
                 { [$style.selected]: selected }
             ]"
+            data-test-id="continue-with-suggestion"
```
Comment (kevinrodrigues) [inline packages/components/molecules/f-searchbox/src/components/formElements/FormFullAddressContinueWithSuggestion.vue:8]: @noorshahJustEat Locator for continue with suggestion component.

**Review by dandel10n — APPROVED** [review]

**Review by simonsandersje — APPROVED** [review]

File: packages/components/molecules/f-searchbox/src/components/formElements/FormSearchField.vue [hunk 2]
```diff
@@ -31,6 +31,12 @@
                 @keydown.down="setKeyboardSuggestion(1)">
 
             <span :class="$style['c-search-placeholder']">{{ copy.fieldPlaceholder }}</span>
+
+            <div
```
Comment (AlixRichards) [inline packages/components/molecules/f-searchbox/src/components/formElements/FormSearchField.vue:35]: I might be remembering wrong, but I thought you had separated the loading spinner into its own component, could we use that here/did I make that up? 😂 🤔

**Review by AlixRichards — APPROVED** [review]
LGTM - just added a question 👍

File: packages/components/molecules/f-searchbox/src/components/formElements/FormSearchField.vue (hunk 2 above)
Comment (kevinrodrigues) [reply packages/components/molecules/f-searchbox/src/components/formElements/FormSearchField.vue:35]: I think maybe it was the SCSS styles I moved to fozzie so it was shareable? at the moment this is pulling from there.

Comment (AlixRichards) [reply packages/components/molecules/f-searchbox/src/components/formElements/FormSearchField.vue:35]: I think you're right - that's what I'm thinking of 👍

**Pull request merged**